Felienne/hedy
# [FEATURE] Adds highscores page

**Description**
In this PR we introduce a new and exciting feature: Highscores! Always wondered how much achievements you have in comparison with other users? Now you know! We create a dedicated highscore page, based on data found on the public profiles table. This table is expanded with the `country` and `achievements_count` value to reduce the amount of DB calls needed on request.

**Fixes**
This PR fixes #2317.

**How to test**
Make sure you are logged in and notice the new "Highscores" option on the profile dropdown menu in the menu bar. Verify that it is all working as expected and that the usernames are click-able to redirect to the public profiles. Also verify that the filtering for world, country and class work as expected. Filtering on class should be hidden when not currently in a class.

![Schermafbeelding 2022-06-23 om 17 10 13](https://user-images.githubusercontent.com/38155424/175333336-5bbd9832-42d6-47a2-9981-34e4472d138d.png)
![Schermafbeelding 2022-06-23 om 17 13 30](https://user-images.githubusercontent.com/38155424/175333487-b918f2c8-44b8-4e09-b9c7-f8a159421ff6.png)
